semantic_search/__init__.py

```diff
@@ -62,7 +62,7 @@ def process_semantic_query(collection, entities_collection, query):
     return {
         "tagged_query": tagged_query,
         "parsed_query": enriched_query,
-        "transformed_query": to_query_string(transformed),
+        "transformed_query": [node["query"] for node in transformed],
```

Comment:
In the manuscript, we renamed `to_query_string` to `to_queries` because we needed to keep the listing. For consistency, we should call `to_queries` here (which has the code on this line in it).

Reply:
This was another slopped change on my end, good catch. I've made all the appropriate changes (including fixing 7.16's code)